src/scss/theme.scss

```diff
@@ -72,6 +74,9 @@
 @import '02_molecules-components/modal';
 @import '02_molecules-components/section';
 @import '02_molecules-components/tooltip';
+@import '02_molecules-components/fixed-cta';
+@import '02_molecules-components/cookie-bar';
+@import '02_molecules-components/chat';
```

Comment:
I'm not sure these are really needed in a skeleton theme since chat provider might differ from affiliate to affiliate.
As far as the cookie bar, I know we use it _pretty much_ everywhere but if we want different colors based on branding, it should be in the child theme.